react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -391,7 +379,7 @@ const ResourceAllocationFormItems: React.FC<
         if (nextImageSelectorType) {
```

Comment:
I think we should add else block with setting values in accelerator value to zero(`0`) considering the image doesn't support AI accelerator. If not, the last page of the session launcher would display AI accelerator value, but when after creating session, the accelerator does not allocated to session, which may confuse user.

Here's the session that I tried to create using simple python kernel.

### Screenshot(s)
- The last page of session launcher (1 Warboy included)
<img width="760" alt="Screenshot 2023-11-15 at 8 15 17 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/dd5475e8-6101-4f55-9add-c9bde46de22d">

- just created session from session launcher in session list
<img width="1021" alt="Screenshot 2023-11-15 at 8 15 44 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/9ea09715-dab4-4256-9e8c-7f3240f07f01">